drivers/ieee802154/ieee802154_uart_pipe.c

```diff
@@ -98,7 +98,7 @@ static u8_t *upipe_rx(u8_t *buf, size_t *off)
 
 static enum ieee802154_hw_caps upipe_get_capabilities(struct device *dev)
 {
-	return IEEE802154_HW_FCS | IEEE802154_HW_2_4_GHZ;
```

Comment:
Keep that one.

At some point the L2 will handle the non-presence of that capability relevantly. In case of uart pipe you just want to fake that it is handled.